.web-docs/components/builder/arm/README.md

```diff
@@ -227,9 +227,9 @@ Providing `temp_resource_group_name` or `location` in combination with
 - `shared_gallery_image_version_end_of_life_date` (string) - The end of life date (2006-01-02T15:04:05.99Z) of the gallery Image Version. This property
   can be used for decommissioning purposes.
 
-- `shared_image_gallery_replica_count` (int32) - The number of replicas of the Image Version to be created per region. This
-  property would take effect for a region when regionalReplicaCount is not specified.
+- `shared_image_gallery_replica_count` (int64) - The number of replicas of the Image Version to be created per region.
   Replica count must be between 1 and 100, but 50 replicas should be sufficient for most use cases.
+  This value can only be 1 when using shallow replication
```

Comment:
```suggestion
  When using shallow replication `use_shallow_replication=true` this value must be set to 1.
```

Reply:
I will re-run make generate to create this suggestion from the config changes